### PR TITLE
Autonomous Constants -> master

### PR DESCRIPTION
**Pull request by hillelv**

### CODE REVIEW

**Review by YHerm — APPROVED** [review]

**Review by Sh-Pe — APPROVED** [review]

File: src/main/java/frc/robot/autonomous/AutonomousConstants.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import com.pathplanner.lib.path.PathConstraints;
+
+
+public class AutonomousConstants {
+
+	public static final String LOG_PATH_PREFIX = "Autonomous/";
+
+	public static final PathConstraints REAL_TIME_CONSTRAINTS = new PathConstraints(2.5, 2.5, 4, 4);
+
+	public static final double CLOSE_TO_TARGET_POSITION_DEADBAND_METERS = 0.5;
```
Comment (drorta) [inline src/main/java/frc/robot/autonomous/AutonomousConstants.java:12]: What is this?

Comment (hillelv) [reply src/main/java/frc/robot/autonomous/AutonomousConstants.java:12]: A constant

Comment (hillelv) [reply src/main/java/frc/robot/autonomous/AutonomousConstants.java:12]: It was in swerve but it is more an auto constant than a swerve constant

Comment (drorta) [reply src/main/java/frc/robot/autonomous/AutonomousConstants.java:12]: Yeah but why do you need this where do you use this what does it mean

Comment (hillelv) [reply src/main/java/frc/robot/autonomous/AutonomousConstants.java:12]: in swerve commands builder if you want to understand more, I'll explain in הכשרת אוטונומים

**Review by drorta — APPROVED** [review]

**Review by NoamRosenberg08 — APPROVED** [review]

**Pull request merged**